### DISCUSSION TITLE
Add NIKAI protocol for THOMSON LCD TV

### DISCUSSION THREAD

**Discussion by Gelip** [Arduino-IRremote/Arduino-IRremote#1258] [Ideas]
Now I use sketch to send NEC code to device using console and my simple remote on breadboard with Arduino Nano:
https://www.elektroda.pl/rtvforum/viewtopic.php?p=19685170#19685170
![remote](https://github.com/user-attachments/assets/601c2455-823f-4841-b87b-f0be02346990)
![sketch_NEC](https://github.com/user-attachments/assets/48200b0f-ee43-4a83-8c94-105c70767f6e)
I need use this my breadboard remote to send code to THOMSON TV in NIKAI protocol.
![THOMSON_NIKAI](https://github.com/user-attachments/assets/1b59a5da-06a3-4515-9bba-c57a4aa43fdd)
I found NIKAI code on this article:
https://conture.by/post/2242
https://conture.by/iresp/IR_DATABASE_TV(STB).pdf
http://conture.by/iresp
This is source **ir_Nikai.cpp** - https://raw.githubusercontent.com/crankyoldgit/IRremoteESP8266/master/src/ir_Nikai.cpp

**Comment by ArminJo** [comment]
NIKAI is no protocol!!!
It is just the mapping of keys to NEC commands!

**Comment by bengtmartensson** [comment]
> NIKAI is no protocol!!!
> It is just the mapping of keys to NEC commands!

Through @Gelip's other posts,

https://github.com/bengtmartensson/IrScrutinizer/discussions/532
http://www.hifi-remote.com/forums/viewtopic.php?t=103521

I get the impression that "NIKAI" is the protocol  that in IrpTransmogrifier/IrScrutinizer and in the JP1 project is known as RCA-38, with another parametrization. It is similar to NEC, but with different timings.
